sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -311,6 +309,35 @@ class HadoopTableReader(
     // Only take the value (skip the key) because Hive works only with values.
     rdd.map(_._2)
   }
+
+  /**
+    * If `spark.sql.hive.fileInputFormat.enabled` is true, this function will optimize the input
```

Comment:
nit. indentation?

Reply:
Something wrong with the code style settings, and fixed!